packages/web-react/src/components/Dropdown/README.md

```diff
@@ -39,7 +40,89 @@ import { Dropdown } from '@lmc-eu/spirit-web-react/components';
 | `trigger['aria-controls']` | `string`                                           | Trigger aria controls   |
 | `trigger.ref`              | `LegacyRef<HTMLButtonElement & HTMLAnchorElement>` | Trigger reference       |
 
+---
+
+# DropdownModern
+
+⚠️ `DropdownModern` component is [deprecated] and will be renamed to `Dropdown` in the next major version.
+This is the React implementation of the `DropdownModern` component.
```

Comment:
This doesn't make sense to me, as there is no DropdownModern component anywhere else. I suggest either removing this sentence at all or 
```suggestion
This is the React implementation of the [`Dropdown`][dropdown] component.
```

Reply:
i wasn't sure about this sentence either, so i removed it.